ui/src/app/views/workspace-share/component.ts

```diff
@@ -6,6 +6,8 @@ import {ErrorHandlingService} from 'app/services/error-handling.service';
 import {isBlank} from 'app/utils';
 
 import {DataAccessLevel} from 'generated';
+import {Profile} from 'generated';
```

Comment:
nit: this import doesn't seem to be directly reference below. Do you need it?

Reply:
Also nit: you could just import these all at once; there's no advantage to multiple import statements.